src/main/webapp/election-info.js

```diff
@@ -320,3 +365,76 @@ Handlebars.registerHelper('findType', function(locationType){
     return "Standard polling location.";
   }
 })
+
+/**
+ * Handlebars helper that processes information from the FVAP API
+ * into a more understandable deadline for the user
+ * 
+ * @param {String} rule the rule for the deadline, which includes
+ *                      how information should be submitted and
+ *                      what the due date is for (postmarked, etc.)
+ * @param {String} votingType the type of action the voter is taking
+ * 
+ * @return {String} a statement summarizing the rule for the user
+ */
+Handlebars.registerHelper('processRule', function(rule, votingType){
+
+  let submissionType = "";
+  let dueDateType =  "";
+
+  if(rule.includes(":")) {
+    let splitRule = rule.split(":");
+    submissionType = splitRule[0].toLowerCase();
+    dueDateType = splitRule[1].toLowerCase();
+  } else {
+    dueDateType = rule.toLowerCase();
+    submissionType = "";
+  }
+
+  if(votingType == "Ballot Return") {
+    votingType = "a ballot";
+  }
+  if(votingType ==   "Ballot Request") {
+    votingType = "a ballot request";
+  }
+  if(votingType == "Registration") {
+    votingType = "voter registration"
+  }
+
+  votingType = votingType.toLowerCase();
```

Comment:
I'm a little confused as what's going on here.. Are you purposely only changing a few of the voting types?

Reply:
I wanted to reformat "Registration" to read "voter registration" for clarity, and add "a" in front of "ballot return" and "ballot request" for grammatical accuracy. I'll clean up the code so that this intention is clearer.

Reply:
I think it would be more clear to just use a switch statement or a map to remap the different voting types. ex:
```
switch(votingType) {
  case "Registration": 
    votingType = "Voter registration";
    break;
  case "Ballot return":
    votingType = "A ballot return";
    break;
  ...
}
```